packages/ember-metal/lib/main.js

```diff
@@ -327,6 +341,23 @@ Ember.isPresent = isPresent;
 
 Ember.merge = merge;
 
+if (Ember.FEATURES.isEnabled('ember-metal-stream')) {
+  Ember.stream = {
+    Stream: Stream,
```

Comment:
Agreed. This is my "fault". I wanted to put everything in `Ember.stream._` because I expect it will become `import _ from "ember-streams"`.